docs/Docker.md

```diff
@@ -1,25 +1,82 @@
 # Using Docker
 
+## Server Initialization
+This was last tested on Ubuntu 24.04 x86
+Note: For this document, macadmins/crypt-server and crypt-server are used interchangeably. macadmins/crypt-server references the downloadedy option and crypt-server represents the local build option. 
+
+``` bash
+git clone https://github.com/grahamgilbert/Crypt-Server.git
+```
+
+Install Docker and Docker Compose plugin following instructions here:
+``` bash
+https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository
+```
+
+Restart the Docker services
+``` bash
+sudo systemctl restart docker
+```
+
+Ensure docker permissions are set. Log out then back in after running this command: 
+``` bash
+sudo usermod -aG docker $USER
+```
+
+We will now install the Docker buildx plugin. 
+``` bash
+sudo apt install docker-buildx
+```
+
+Verify plugin state: 
+``` bash
+docker info | head -n8
+```
+
+Response should be similar to: 
+``` bash
+Client:
+ Version:    24.0.7
+ Context:    default
+ Debug Mode: false
+ Plugins:
+  buildx: Docker Buildx (Docker Inc.)
+    Version:  0.12.1
+    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
+```
+
+Build the Dockerfile:
+``` bash
+docker buildx build -t crypt-server .
```

Comment:
Is this necessary for most people? I would prefer the docs to reference the built image.

Reply:
Will remove